Error when commit message includes { or } for 5.6.4+

We're getting errors in our build pipeline after merging a PR with a { in the name. This is resolved by downgrading GitVersion to 5.6.3.

I would guess it is related to [this code](https://github.com/GitTools/actions/blob/21f970525bdf92932571d0a9d0fa51a2404df2fa/src/tasks/gitversion/main.ts#L45).

Error message (Azure DevOps):
```
##[error]SyntaxError: Unexpected token S in JSON at position 1
```

Output from 5.6.6:
```
INFO [03/19/21 16:39:20:93] Working directory: C:\Code\src
INFO [03/19/21 16:39:20:97] Project root is: C:\Code\src\
INFO [03/19/21 16:39:20:97] DotGit directory is: C:\Code\src\.git
INFO [03/19/21 16:39:21:01] Begin: Loading version variables from disk cache
  INFO [03/19/21 16:39:21:01] Cache file C:\Code\src\.git\gitversion_cache\60B67F852C73D243B79D5D48A01600B3732665D7.yml not found.
  INFO [03/19/21 16:39:21:01] End: Loading version variables from disk cache (Took: 2.69ms)
  INFO [03/19/21 16:39:21:12] Using latest commit on specified branch
  INFO [03/19/21 16:39:21:12] No branch configuration found for branch laje/S-77070-aas-tagging, falling back to default configuration
  INFO [03/19/21 16:39:21:13] Begin: Attempting to inherit branch configuration from parent branch
    INFO [03/19/21 16:39:21:15] Begin: Finding branch source of 'laje/S-77070-aas-tagging'
      INFO [03/19/21 16:39:21:16] Begin: Finding merge base between 'laje/S-77070-aas-tagging' and 'master'.
        INFO [03/19/21 16:39:21:17] Found merge base of 96e77c5 Merged PR 1685: {S-77173] Update to tagging of the Resource Groups with Deployment Id and Blueprint version
        INFO [03/19/21 16:39:21:18] Merge base of laje/S-77070-aas-tagging' and 'master is 96e77c5 Merged PR 1685: {S-77173] Update to tagging of the Resource Groups with Deployment Id and Blueprint version
      INFO [03/19/21 16:39:21:18] End: Finding merge base between 'laje/S-77070-aas-tagging' and 'master'. (Took: 22.00ms)
      INFO [03/19/21 16:39:21:18] Begin: Finding merge base between 'laje/S-77070-aas-tagging' and 'origin/master'.
        INFO [03/19/21 16:39:21:19] Found merge base of 96e77c5 Merged PR 1685: {S-77173] Update to tagging of the Resource Groups with Deployment Id and Blueprint version
        INFO [03/19/21 16:39:21:19] Merge base of laje/S-77070-aas-tagging' and 'origin/master is 96e77c5 Merged PR 1685: {S-77173] Update to tagging of the Resource Groups with Deployment Id and Blueprint version
      INFO [03/19/21 16:39:21:19] End: Finding merge base between 'laje/S-77070-aas-tagging' and 'origin/master'. (Took: 1.91ms)
      INFO [03/19/21 16:39:21:19] Multiple source branches have been found, picking the first one (master).
This may result in incorrect commit counting.
Options were:
master, origin/master
    INFO [03/19/21 16:39:21:19] End: Finding branch source of 'laje/S-77070-aas-tagging' (Took: 45.28ms)
    INFO [03/19/21 16:39:21:20] Begin: Getting branches containing the commit '96e77c5'.
      INFO [03/19/21 16:39:21:20] Trying to find direct branches.
      INFO [03/19/21 16:39:21:20] No direct branches found, searching through all branches.
      INFO [03/19/21 16:39:21:20] Searching for commits reachable from 'master'.
      INFO [03/19/21 16:39:21:20] The branch 'master' has a matching commit.
      INFO [03/19/21 16:39:21:20] Searching for commits reachable from 'origin/master'.
      INFO [03/19/21 16:39:21:20] The branch 'origin/master' has a matching commit.
    INFO [03/19/21 16:39:21:20] End: Getting branches containing the commit '96e77c5'. (Took: 3.73ms)
    INFO [03/19/21 16:39:21:20] Begin: Getting branches containing the commit 'eb45d5c'.
      INFO [03/19/21 16:39:21:20] Trying to find direct branches.
      INFO [03/19/21 16:39:21:20] No direct branches found, searching through all branches.
      INFO [03/19/21 16:39:21:20] Searching for commits reachable from 'master'.
      INFO [03/19/21 16:39:21:24] The branch 'master' has no matching commits.
      INFO [03/19/21 16:39:21:24] Searching for commits reachable from 'origin/master'.
      INFO [03/19/21 16:39:21:28] The branch 'origin/master' has no matching commits.
    INFO [03/19/21 16:39:21:28] End: Getting branches containing the commit 'eb45d5c'. (Took: 79.42ms)
    INFO [03/19/21 16:39:21:28] Found possible parent branches: master, origin/master
    WARN [03/19/21 16:39:21:29] Failed to inherit Increment branch configuration, ended up with: master, origin/master
Falling back to master branch config
  INFO [03/19/21 16:39:21:29] End: Attempting to inherit branch configuration from parent branch (Took: 157.70ms)
  INFO [03/19/21 16:39:21:37] Running against branch: laje/S-77070-aas-tagging (eb45d5c Remove displayName tag from AAS to ensure version tags are not overwritten by blueprints)
  INFO [03/19/21 16:39:21:38] Begin: Calculating base versions
    INFO [03/19/21 16:39:21:44] Fallback base version: 0.1.0 with commit count source 36d3335c1dea8313ddd21c5b5959c84fdcd428c8
    INFO [03/19/21 16:39:21:76] Git tag '0.1.318': 0.1.318 with commit count source 0fec4df892ca3b0b0f7e7a002df7acf1cf0d3aba
    INFO [03/19/21 16:39:21:76] Git tag '0.1.316': 0.1.316 with commit count source a2cf1de543a0f6908510a2c17976f91289d093b4
    INFO [03/19/21 16:39:21:76] Git tag '0.1.314': 0.1.314 with commit count source 98cd5967bb3f35502ef90e127a65c162ff897c30
    INFO [03/19/21 16:39:21:76] Git tag '0.1.312': 0.1.312 with commit count source 0507389f2e09919be39a7b4a5c1429848f875017
    INFO [03/19/21 16:39:21:76] Git tag '0.1.311': 0.1.311 with commit count source 4b9a14a942a07c22df28db10ba497cd513e1cbb1
    INFO [03/19/21 16:39:21:76] Git tag '0.1.310': 0.1.310 with commit count source 2e7f3c4a7a6dd9055f7bc4b328be3597fb12a942
    INFO [03/19/21 16:39:21:76] Git tag '0.1.309': 0.1.309 with commit count source 245c704e5cfa20daa7ace4bcfe61e1457f1f185c
    INFO [03/19/21 16:39:21:76] Git tag '0.1.308': 0.1.308 with commit count source 4814f7129771338075929d96d24860c6ea5859df
    INFO [03/19/21 16:39:21:77] Git tag '0.1.307': 0.1.307 with commit count source 440431d245096f22ed1436a49a7b74b2ee99fdfb
    INFO [03/19/21 16:39:21:77] Git tag '0.1.306': 0.1.306 with commit count source c9b82cec4f4cbb4817382f1a16b6d33a5c5a1bc0
    INFO [03/19/21 16:39:21:77] Git tag '0.1.305': 0.1.305 with commit count source b5dee1fbca0a1ae8844b60ff3f198d1277bcfdc9
    INFO [03/19/21 16:39:21:77] Git tag '0.1.304': 0.1.304 with commit count source ff69141966b4c168861e1ce7a225db11b76e2660
    INFO [03/19/21 16:39:21:77] Git tag '0.1.303': 0.1.303 with commit count source 54246f06cb45bcff8a9e5213ffd4d2b5d9e7fe1b
    INFO [03/19/21 16:39:21:77] Git tag '0.1.301': 0.1.301 with commit count source ae21607ea2b881fda7e56505c9dfc3df0cb57aa1
    INFO [03/19/21 16:39:21:77] Git tag '0.1.300': 0.1.300 with commit count source af9c10db3efc5a377a6500b798a6b20557113d4d
    INFO [03/19/21 16:39:21:77] Git tag '0.1.297': 0.1.297 with commit count source 3f6337624f74816df5b9f1e35bace0b0b39f6e46
    INFO [03/19/21 16:39:21:77] Git tag '0.1.296': 0.1.296 with commit count source 8c33e21a6c56b65ab12e0999d906a691e695c081
    INFO [03/19/21 16:39:21:77] Git tag '0.1.295': 0.1.295 with commit count source 0fa29d1263969e9ca2c0fa8fcf928b76db3cbc3b
    INFO [03/19/21 16:39:21:77] Git tag '0.1.294': 0.1.294 with commit count source 13274a99ef4292001e06bc17dcd92d33960ff460
    INFO [03/19/21 16:39:21:77] Git tag '0.1.293': 0.1.293 with commit count source 4edda1441e8e2a8ee275a209f5fed3c9f3a972f3
    INFO [03/19/21 16:39:21:77] Git tag '0.1.292': 0.1.292 with commit count source c168efc5e05f7f6c1f13951b50f4d24c9f95027f
    INFO [03/19/21 16:39:21:77] Git tag '0.1.291': 0.1.291 with commit count source a31a1352bca74815432fcd5901d4a4e1eddbf4d9
    INFO [03/19/21 16:39:21:77] Git tag '0.1.290': 0.1.290 with commit count source 54aa7b21c3e9e1ef4b6df13c255f1610e5d0027e
    INFO [03/19/21 16:39:21:77] Git tag '0.1.289': 0.1.289 with commit count source f8b3e1aa09633e97a4bd28c37e9c51e998809d53
    INFO [03/19/21 16:39:21:77] Git tag '0.1.287': 0.1.287 with commit count source 6ac1bb2e0094d626d0b0d76821c1cc1b9f0d062d
    INFO [03/19/21 16:39:21:77] Git tag '0.1.286': 0.1.286 with commit count source f518b269246bfcec233ee3ac3c8e01053fda365e
    INFO [03/19/21 16:39:21:77] Git tag '0.1.285': 0.1.285 with commit count source 82ab2180946d8f9880b3afbceca9a08e9200e6d2
    INFO [03/19/21 16:39:21:77] Git tag '0.1.283': 0.1.283 with commit count source 73517604ff01f2fa4e25f51f8d3c3064cea571ba
    INFO [03/19/21 16:39:21:77] Git tag '0.1.282': 0.1.282 with commit count source f3fb529afd66bfa402ab2c173327fd6f36eebd76
    INFO [03/19/21 16:39:21:77] Git tag '0.1.281': 0.1.281 with commit count source 117ef39b46c9f517ebe2f02c35101751ccac96b6
    INFO [03/19/21 16:39:21:77] Git tag '0.1.280': 0.1.280 with commit count source 51e5b38a36f6e935c9c070af13cb0ca62f64b6b8
    INFO [03/19/21 16:39:21:77] Git tag '0.1.279': 0.1.279 with commit count source 32f7776e9d1e01d0e40641da14631a53b90e36f0
    INFO [03/19/21 16:39:21:77] Git tag '0.1.278': 0.1.278 with commit count source 7f5e5ebb8a4514e2ab2fa7186501334bf7c4ff1e
    INFO [03/19/21 16:39:21:77] Git tag '0.1.277': 0.1.277 with commit count source 55192aa48833d9f584c34fc171fb8ac20a6f2191
    INFO [03/19/21 16:39:21:77] Git tag '0.1.274': 0.1.274 with commit count source 7f4683b89a23e80ea90d842fa2a6e798ca65ad3b
    INFO [03/19/21 16:39:21:77] Git tag '0.1.273': 0.1.273 with commit count source 8bd7af6c30de42cd9ffd3119f48a9b5970a2a9cd
    INFO [03/19/21 16:39:21:77] Git tag '0.1.272': 0.1.272 with commit count source 88f58ca71e5a7a7d3bdd2f388212a339aa0f7916
    INFO [03/19/21 16:39:21:77] Git tag '0.1.271': 0.1.271 with commit count source 3302bc0a6410e2bbef8cd06a9bca88a6302e2f2f
    INFO [03/19/21 16:39:21:77] Git tag '0.1.270': 0.1.270 with commit count source c136fea85471965c9324ed03b0d667326bbdc989
    INFO [03/19/21 16:39:21:77] Git tag '0.1.268': 0.1.268 with commit count source 31c616510b1ad4073d70a621d1b57c327a2d3ec7
    INFO [03/19/21 16:39:21:77] Git tag '0.1.267': 0.1.267 with commit count source 4e6b0cdfdfd77e4e5d36c46288a5b3ec1623a11d
    INFO [03/19/21 16:39:21:77] Git tag '0.1.266': 0.1.266 with commit count source a46988e458a561cb21e2d49906dc486ff8c4422d
    INFO [03/19/21 16:39:21:77] Git tag '0.1.265': 0.1.265 with commit count source a8e30c26fb0f9ae9796a35c8d2b0e6edb5b8a9f3
    INFO [03/19/21 16:39:21:77] Git tag '0.1.264': 0.1.264 with commit count source ce45a6484324ab2c5b4ad03dce46748145dd3a53
    INFO [03/19/21 16:39:21:77] Git tag '0.1.263': 0.1.263 with commit count source 44a1b58694bbabd17e13aedab317d1c68f775415
    INFO [03/19/21 16:39:21:78] Git tag '0.1.262': 0.1.262 with commit count source 32d0a20fdc016b17e0fb7ce09e886eba47b2d4b9
    INFO [03/19/21 16:39:21:78] Git tag '0.1.260': 0.1.260 with commit count source 76a24bf39a759be8f108c9bea5737435c3a0843f
    INFO [03/19/21 16:39:21:78] Git tag '0.1.259': 0.1.259 with commit count source bb060e2e2a607e72d2df364538b34fa69b1d8c47
    INFO [03/19/21 16:39:21:78] Git tag '0.1.258': 0.1.258 with commit count source d4e826ba55346fe823baafb2fed5ab150ac05457
    INFO [03/19/21 16:39:21:78] Git tag '0.1.256': 0.1.256 with commit count source 3e8b032dc09aa5f5efc1f9da63b242fdd948a136
    INFO [03/19/21 16:39:21:78] Git tag '0.1.255': 0.1.255 with commit count source 3b0ec26ba80615852377828baa38d306abe80606
    INFO [03/19/21 16:39:21:78] Git tag '0.1.254': 0.1.254 with commit count source 50025f48e9c510e242c13b85b264459ff77800ae
    INFO [03/19/21 16:39:21:78] Git tag '0.1.253': 0.1.253 with commit count source 5131c29fddf80d8ff05543673feada055f74d7e1
    INFO [03/19/21 16:39:21:78] Git tag '0.1.252': 0.1.252 with commit count source 07201a59d40ae0cfd6623a314c18b36b95d08e82
    INFO [03/19/21 16:39:21:78] Git tag '0.1.251': 0.1.251 with commit count source 2ad3c0979240f35c09a806b56706d046fad9f0bc
    INFO [03/19/21 16:39:21:78] Git tag '0.1.250': 0.1.250 with commit count source 2fd5c817068d754f8d18e1a8f94e520b05e64771
    INFO [03/19/21 16:39:21:78] Git tag '0.1.247': 0.1.247 with commit count source 01635c4b62702fac70990099d94a7b4eb1cd5213
    INFO [03/19/21 16:39:21:78] Git tag '0.1.246': 0.1.246 with commit count source e28a3bae0feb144b78b6351d2ed7d06935df5658
    INFO [03/19/21 16:39:21:78] Git tag '0.1.245': 0.1.245 with commit count source 8f662a50efaacd1cccfcc95195fae12fd2eaf346
    INFO [03/19/21 16:39:21:78] Git tag '0.1.242': 0.1.242 with commit count source 8a061f663d1bb6a7ca0c4cfe0745247da4a25ae1
    INFO [03/19/21 16:39:21:78] Git tag '0.1.240': 0.1.240 with commit count source e82ec1194dd1848f92fab2aa4c06fd183978796c
    INFO [03/19/21 16:39:21:78] Git tag '0.1.239': 0.1.239 with commit count source 374024a47f95fe92d8092f4ff77e8a307f121c2e
    INFO [03/19/21 16:39:21:78] Git tag '0.1.238': 0.1.238 with commit count source cb8e7c4d87e16a7d334c163ea4e226a3dce50565
    INFO [03/19/21 16:39:21:78] Git tag '0.1.237': 0.1.237 with commit count source fff2c95142dd5031a735a78667e1d52bd12dae18
    INFO [03/19/21 16:39:21:78] Git tag '0.1.236': 0.1.236 with commit count source 24de1014bceb2877ab716114b6bccce3e75a74b9
    INFO [03/19/21 16:39:21:78] Git tag '0.1.235': 0.1.235 with commit count source 376b4049f97e40e06044f30a0e1920d060089268
    INFO [03/19/21 16:39:21:78] Git tag '0.1.234': 0.1.234 with commit count source ad126ee3dd3da577ee70697a2f95183decf6dc34
    INFO [03/19/21 16:39:21:78] Git tag '0.1.233': 0.1.233 with commit count source 48d1b1248865a285a404605b0808940c5b7aa3b6
    INFO [03/19/21 16:39:21:78] Git tag '0.1.230': 0.1.230 with commit count source 0db5dc15e4451411a7d98d04f07fcb66207dbeed
    INFO [03/19/21 16:39:21:78] Git tag '0.1.229': 0.1.229 with commit count source effa4ea2c3d5ada3bb000b3d963fd0228777e80a
    INFO [03/19/21 16:39:21:78] Git tag '0.1.228': 0.1.228 with commit count source f7baa26f69562b031078c46b725466ca170f5cc3
    INFO [03/19/21 16:39:21:78] Git tag '0.1.227': 0.1.227 with commit count source a335d8365a5250bbdff0fa0a03c5f114cc7bfc3b
    INFO [03/19/21 16:39:21:78] Git tag '0.1.226': 0.1.226 with commit count source 5b3490e994aaf8016a7f776864525c6129a3ee5e
    INFO [03/19/21 16:39:21:78] Git tag '0.1.225': 0.1.225 with commit count source fa381ea8dfb07fdc8457c5eff7f6868898de8660
    INFO [03/19/21 16:39:21:78] Git tag '0.1.224': 0.1.224 with commit count source 50fa320797042432b05039204b2382b4143cc54a
    INFO [03/19/21 16:39:21:78] Git tag '0.1.223': 0.1.223 with commit count source 94f05ad4db4d52e20fdf54cfc9be1cd297cc8e66
    INFO [03/19/21 16:39:21:78] Git tag '0.1.222': 0.1.222 with commit count source ac24a1ea69dce68fc327335bf4f3292dd372c70f
    INFO [03/19/21 16:39:21:78] Git tag '0.1.220': 0.1.220 with commit count source 5b0e5ed36932c7d0e3bd01d4f779daa8cf33c7df
    INFO [03/19/21 16:39:21:78] Git tag '0.1.219': 0.1.219 with commit count source b7d96a0175d08ee98e26ac9d671a145ad80ccf2b
    INFO [03/19/21 16:39:21:78] Git tag '0.1.217': 0.1.217 with commit count source 5b04d032f0f3969c0d52c99eddc2b22b8e2bc5be
    INFO [03/19/21 16:39:21:78] Git tag '0.1.216': 0.1.216 with commit count source 10916ab86054b5ae2b40143bb7c9368cd48aab58
    INFO [03/19/21 16:39:21:78] Git tag '0.1.215': 0.1.215 with commit count source 01b0b84aaa0de740d6adc55c3b5510568cc1793d
    INFO [03/19/21 16:39:21:79] Git tag '0.1.214': 0.1.214 with commit count source 846c5eec5200faad75973eb93f77d38b765c647a
    INFO [03/19/21 16:39:21:79] Git tag '0.1.213': 0.1.213 with commit count source 6e6ecb779e2c314913071118ecc933d0744b944b
    INFO [03/19/21 16:39:21:79] Git tag '0.1.212': 0.1.212 with commit count source 80203bed69e0cbc4c193f5bb91561bc27865097f
    INFO [03/19/21 16:39:21:79] Git tag '0.1.211': 0.1.211 with commit count source 3f0aed306fecc136f1b925da8cf7c8896aaa36ce
    INFO [03/19/21 16:39:21:79] Git tag '0.1.208': 0.1.208 with commit count source b1b26226b7aaeb6a767f0aefcfc736292731c4f9
    INFO [03/19/21 16:39:21:79] Git tag '0.1.207': 0.1.207 with commit count source 8c11e646a274f02e82c86de4f8ec17692afd2384
    INFO [03/19/21 16:39:21:79] Git tag '0.1.206': 0.1.206 with commit count source 1565759911c3fde12d390d0a59c759690ddec7bc
    INFO [03/19/21 16:39:21:79] Git tag '0.1.205': 0.1.205 with commit count source f874f6565b08b97919d75172b3e5a1e164cf90ed
    INFO [03/19/21 16:39:21:79] Git tag '0.1.204': 0.1.204 with commit count source 826a43a44af44947983a4cb600a5030716cc9804
    INFO [03/19/21 16:39:21:79] Git tag '0.1.203': 0.1.203 with commit count source 0abad3c83f9e63e6cae6dc763cc513ee2143b7a0
    INFO [03/19/21 16:39:21:79] Git tag '0.1.202': 0.1.202 with commit count source aedc8ee9b75d6581b1ddeb9bd3bdc8a4fa801f18
    INFO [03/19/21 16:39:21:79] Git tag '0.1.201': 0.1.201 with commit count source 671ddc16ee9360f8aa8eb8a3d0a7ffc21785b4e1
    INFO [03/19/21 16:39:21:79] Git tag '0.1.200': 0.1.200 with commit count source 3d41e296bee5c5dc3b2aca6065d21d6f62e54b26
    INFO [03/19/21 16:39:21:79] Git tag '0.1.199': 0.1.199 with commit count source 6f4dc6bba2b7c21c5d1b92e6a7f0390196d77b4d
    INFO [03/19/21 16:39:21:79] Git tag '0.1.198': 0.1.198 with commit count source bcf70bfe4d7ace18ea2a858831548a1824039131
    INFO [03/19/21 16:39:21:79] Git tag '0.1.197': 0.1.197 with commit count source 7ee532f1ee04fc34c7696d46795ccc87fe0d0baf
    INFO [03/19/21 16:39:21:79] Git tag '0.1.196': 0.1.196 with commit count source 99421a9da93990c0a7f7cf6f31996b3126c99423
    INFO [03/19/21 16:39:21:79] Git tag '0.1.195': 0.1.195 with commit count source 908146987db4359dde01346fdc5139b6b7ee39d0
    INFO [03/19/21 16:39:21:79] Git tag '0.1.194': 0.1.194 with commit count source 2cabaf5e5335ce5fa50fa180b197c79a67c966c1
    INFO [03/19/21 16:39:21:79] Git tag '0.1.193': 0.1.193 with commit count source 84d7206e405dc1f53208cc8f0869a53e979c9046
    INFO [03/19/21 16:39:21:79] Git tag '0.1.192': 0.1.192 with commit count source 31a7b6ac1771a042708f154c30dab200c65c7ba6
    INFO [03/19/21 16:39:21:79] Git tag '0.1.191': 0.1.191 with commit count source 432435ed4739f46353e2d99e55c32fb123b59b7e
    INFO [03/19/21 16:39:21:79] Git tag '0.1.190': 0.1.190 with commit count source a7a0779e3555695e4d27f5312af67268749919a1
    INFO [03/19/21 16:39:21:79] Git tag '0.1.189': 0.1.189 with commit count source ea72206de480d90176180d6c4d48fb16a5715e1b
    INFO [03/19/21 16:39:21:79] Git tag '0.1.188': 0.1.188 with commit count source 616f70696d5bf403535042cff794a9f4d78c2ed8
    INFO [03/19/21 16:39:21:79] Git tag '0.1.187': 0.1.187 with commit count source c37a445a10727c0fa040499cba790911e97ada12
    INFO [03/19/21 16:39:21:79] Git tag '0.1.186': 0.1.186 with commit count source 9122d7e50f0787a3550ea23656b035408132cfdb
    INFO [03/19/21 16:39:21:79] Git tag '0.1.185': 0.1.185 with commit count source e3c74fffb055731823ea1e8ca46fe3cf10b91e5e
    INFO [03/19/21 16:39:21:79] Git tag '0.1.184': 0.1.184 with commit count source 0f3ac66602454ca8cd768285f223a86fe46dc200
    INFO [03/19/21 16:39:21:79] Git tag '0.1.183': 0.1.183 with commit count source 304a4bdf920bfddd00dc6a1fcf6d51fa85fa81ab
    INFO [03/19/21 16:39:21:79] Git tag '0.1.182': 0.1.182 with commit count source 240d3b36bfde5f58d6e668e88f86ea341cb447e9
    INFO [03/19/21 16:39:21:79] Git tag '0.1.180': 0.1.180 with commit count source 71bf1d39b53f16e02a6e78b5904b4cbf22c9a165
    INFO [03/19/21 16:39:21:79] Git tag '0.1.179': 0.1.179 with commit count source 01306fe508a133b379f8f5d0fa47f91be526533b
    INFO [03/19/21 16:39:21:79] Git tag '0.1.178': 0.1.178 with commit count source eee2c0c3512384b370d65cb8ba1cc1ff7d19cd32
    INFO [03/19/21 16:39:21:79] Git tag '0.1.177': 0.1.177 with commit count source 5552bfce84f7404c303d41740b62dfd38f685def
    INFO [03/19/21 16:39:21:79] Git tag '0.1.176': 0.1.176 with commit count source 5f031301adefeec67490f4999633ebd8df11bb57
    INFO [03/19/21 16:39:21:79] Git tag '0.1.175': 0.1.175 with commit count source a9961593d49020af1d8dd2156f0d4e39eef59ca0
    INFO [03/19/21 16:39:21:79] Git tag '0.1.174': 0.1.174 with commit count source 20059f70882655edd913206f4357f5bf10c0d1c3
    INFO [03/19/21 16:39:21:80] Git tag '0.1.173': 0.1.173 with commit count source 8c630ebbd7dd5a43d7289be30a64a2ac9104e49c
    INFO [03/19/21 16:39:21:80] Git tag '0.1.171': 0.1.171 with commit count source 1d44590b9b283a0f7d8f9fa6ddddc1150709bb91
    INFO [03/19/21 16:39:21:80] Git tag '0.1.170': 0.1.170 with commit count source 95a1d162504c797706cb30941df1b6daa4c4d9af
    INFO [03/19/21 16:39:21:80] Git tag '0.1.169': 0.1.169 with commit count source 0bbf76a6e9752e2624256c614bec2ecc1cd70a9e
    INFO [03/19/21 16:39:21:80] Git tag '0.1.168': 0.1.168 with commit count source 2a3abc900b1f2d974b569cdb825e3fe1b763d51f
    INFO [03/19/21 16:39:21:80] Git tag '0.1.167': 0.1.167 with commit count source af7431415fce08f88db32eba666e81fe88dc9923
    INFO [03/19/21 16:39:21:80] Git tag '0.1.165': 0.1.165 with commit count source ff71b6dfc7061dc684adf9b5cd855224453402a1
    INFO [03/19/21 16:39:21:80] Git tag '0.1.164': 0.1.164 with commit count source 62f002d11e26bff506489464a7fab7e023182344
    INFO [03/19/21 16:39:21:80] Git tag '0.1.163': 0.1.163 with commit count source 3f0d37ffa52bdbd1598ee658d6b6448590127938
    INFO [03/19/21 16:39:21:80] Git tag '0.1.161': 0.1.161 with commit count source 3e8f48ebc43caaaa1c7243a50fc802e3b66a2e89
    INFO [03/19/21 16:39:21:80] Git tag '0.1.160': 0.1.160 with commit count source 779450667485aabf8a64f35d544fe284640aaa3b
    INFO [03/19/21 16:39:21:80] Git tag '0.1.159': 0.1.159 with commit count source 654e113393f9d35424002bab482cb6f41394c529
    INFO [03/19/21 16:39:21:80] Git tag '0.1.158': 0.1.158 with commit count source 9c797ff3917922bf0a70855657a29ffe44d9ef93
    INFO [03/19/21 16:39:21:80] Git tag '0.1.157': 0.1.157 with commit count source e34043606bbd9b0bc29e6f02145dfb46cfa6cf02
    INFO [03/19/21 16:39:21:80] Git tag '0.1.156': 0.1.156 with commit count source 612ec318e43d523a22c875ddd3369671ce1509e9
    INFO [03/19/21 16:39:21:80] Git tag '0.1.154': 0.1.154 with commit count source 3805b1b7367e3106a1080ffc1965794bffd8929a
    INFO [03/19/21 16:39:21:80] Git tag '0.1.153': 0.1.153 with commit count source 0cd248f758e1ecf094c0fd56f1cda939b65b5265
    INFO [03/19/21 16:39:21:80] Git tag '0.1.152': 0.1.152 with commit count source 236fcbffe02d616fe3b77fe26a377a22fb780bb8
    INFO [03/19/21 16:39:21:80] Git tag '0.1.151': 0.1.151 with commit count source ebdfd3fb1e8235462d445a6d966c1fb6af9c5b89
    INFO [03/19/21 16:39:21:80] Git tag '0.1.150': 0.1.150 with commit count source 5a073f104ff93a3d3b007515967a8d3d2be2dd05
    INFO [03/19/21 16:39:21:80] Git tag '0.1.149': 0.1.149 with commit count source 9903785fea9912e83bf1133a3ff1f30462816307
    INFO [03/19/21 16:39:21:80] Git tag '0.1.148': 0.1.148 with commit count source ee652a26ef8a75d4b3a006d063b2c6b1e0f1b6d5
    INFO [03/19/21 16:39:21:80] Git tag '0.1.147': 0.1.147 with commit count source 7f60a6dacd29a388b94d451c86def628e6cd8b68
    INFO [03/19/21 16:39:21:80] Git tag '0.1.146': 0.1.146 with commit count source 043b2c1c4e244788183023b775c52a52193a6498
    INFO [03/19/21 16:39:21:80] Git tag '0.1.145': 0.1.145 with commit count source b774dca8a6d939054938846288341496b1e293b4
    INFO [03/19/21 16:39:21:80] Git tag '0.1.142': 0.1.142 with commit count source ad2f4153ca86b39c07852504cc7376d89678f446
    INFO [03/19/21 16:39:21:80] Git tag '0.1.138': 0.1.138 with commit count source 223b277438d9755484420dace3c6cb971bbea196
    INFO [03/19/21 16:39:21:80] Git tag '0.1.136': 0.1.136 with commit count source f7e64fa2f4ead3dcb04cde305fc420064a731025
    INFO [03/19/21 16:39:21:80] Git tag '0.1.135': 0.1.135 with commit count source 4ab6aeb957f63d40a1d2e33aed3eb45525c56787
    INFO [03/19/21 16:39:21:80] Git tag '0.1.132': 0.1.132 with commit count source 4d71ede22ab287288f1d3a5ad27d14834fcd426e
    INFO [03/19/21 16:39:21:80] Git tag '0.1.131': 0.1.131 with commit count source 9982e1d8f67a5ca9835496d785141882dc0a629c
    INFO [03/19/21 16:39:21:80] Git tag '0.1.130': 0.1.130 with commit count source b28a7626bbbd6a3f600abef435447dbb2fc2bf3c
    INFO [03/19/21 16:39:21:80] Git tag '0.1.129': 0.1.129 with commit count source c9db39a1a095fa3f6e59e9f72e42dc435c74f901
    INFO [03/19/21 16:39:21:80] Git tag '0.1.128': 0.1.128 with commit count source 395a4f41434e3d9dd00b0e963045043fcd9d0ac0
    INFO [03/19/21 16:39:21:80] Git tag '0.1.126': 0.1.126 with commit count source ece01e1aedf85ae8fe8b24b4e726b44c1d5a3f47
    INFO [03/19/21 16:39:21:80] Git tag '0.1.125': 0.1.125 with commit count source 7ecfd33f459cc96db57aaee09da61c6492b24d5e
    INFO [03/19/21 16:39:21:80] Git tag '0.1.124': 0.1.124 with commit count source 462bc1dff73db75efa49e0215ae2290bd431d676
    INFO [03/19/21 16:39:21:80] Git tag '0.1.123': 0.1.123 with commit count source f842f0f8e8edc5d470c68afb2e1137a5bf380456
    INFO [03/19/21 16:39:21:80] Git tag '0.1.122': 0.1.122 with commit count source 1044b12226ec683adef4564f8d013c4ac82a84e0
    INFO [03/19/21 16:39:21:80] Git tag '0.1.121': 0.1.121 with commit count source 7d110cbaef6bf24047defda0740bc2e9ee7c1da7
    INFO [03/19/21 16:39:21:81] Git tag '0.1.120': 0.1.120 with commit count source 4a002e65bc87a129ec948a9ec94ba66218afb3a3
    INFO [03/19/21 16:39:21:81] Git tag '0.1.119': 0.1.119 with commit count source 37513f1e0106b89a254241eb2e7edbd2c6008df2
    INFO [03/19/21 16:39:21:81] Git tag '0.1.118': 0.1.118 with commit count source 9c6123bd3d2e333ce68d6b8bda29a55b998ff887
    INFO [03/19/21 16:39:21:81] Git tag '0.1.116': 0.1.116 with commit count source c55619a21f5c03c8fbbf1bf7615b7d3e317b759e
    INFO [03/19/21 16:39:21:81] Git tag '0.1.115': 0.1.115 with commit count source 312062d88d8254f3528e4890af333eb71c0b2703
    INFO [03/19/21 16:39:21:81] Git tag '0.1.114': 0.1.114 with commit count source c5a527aff049d26bd95bb446bcd079494f4bfc66
    INFO [03/19/21 16:39:21:81] Git tag '0.1.113': 0.1.113 with commit count source a41bbe9ddaaab41f565088af2a73a0cfafe919a5
    INFO [03/19/21 16:39:21:81] Git tag '0.1.112': 0.1.112 with commit count source 7f0246f57ed9cb4a889f28cbcf7f27b37209ce37
    INFO [03/19/21 16:39:21:81] Git tag '0.1.110': 0.1.110 with commit count source 4f864fcce77f321b40df514dafff0b4b7aa571f5
    INFO [03/19/21 16:39:21:81] Git tag '0.1.108': 0.1.108 with commit count source 557fed1ba69221d2676f1699dd283ecfbfde5f3e
    INFO [03/19/21 16:39:21:81] Git tag '0.1.107': 0.1.107 with commit count source dc4d9b528b6cfe20f5ff0dee2da224c1ee4fe17a
    INFO [03/19/21 16:39:21:81] Git tag '0.1.106': 0.1.106 with commit count source b7eec87b90c8f63b7b8b7b11b06684002b9fe772
    INFO [03/19/21 16:39:21:81] Git tag '0.1.105': 0.1.105 with commit count source 52130778e640179c00a096e0b53658f0d8c59860
    INFO [03/19/21 16:39:21:81] Git tag '0.1.104': 0.1.104 with commit count source c8612fd5d343162bfe0e1dbaa08c2f53b6c0f7bf
    INFO [03/19/21 16:39:21:81] Git tag '0.1.103': 0.1.103 with commit count source d97a7b6175d0ef137076e7f9dd15450baac6ed42
    INFO [03/19/21 16:39:21:81] Git tag '0.1.100': 0.1.100 with commit count source 99c4e5d3eb4439314a369cff17cc3b0cf048244c
    INFO [03/19/21 16:39:21:81] Git tag '0.1.99': 0.1.99 with commit count source a504ddd5395db7ac7b376b35df1329d5a608f0d0
    INFO [03/19/21 16:39:21:81] Git tag '0.1.98': 0.1.98 with commit count source a6306f961fa73f58013bfc6a8ac83f754df1a38a
    INFO [03/19/21 16:39:21:81] Git tag '0.1.97': 0.1.97 with commit count source bfdc1e25078078fa52d59a93d73242f32a0e5c23
    INFO [03/19/21 16:39:21:81] Git tag '0.1.96': 0.1.96 with commit count source 4cb4b236b62e0eb9b50abff7f8e5e78fe24385e4
    INFO [03/19/21 16:39:21:81] Git tag '0.1.95': 0.1.95 with commit count source 3967a52effb5b2fa82f2b344e853ff9169dceec1
    INFO [03/19/21 16:39:21:81] Git tag '0.1.93': 0.1.93 with commit count source 61a316b67b5d11325f4234bfcf098b369027bea1
    INFO [03/19/21 16:39:21:81] Git tag '0.1.92': 0.1.92 with commit count source 0ae1ce0816fd785741bbde797177049c7c133f60
    INFO [03/19/21 16:39:21:81] Git tag '0.1.90': 0.1.90 with commit count source 1b8cd20148bcd6054b87a3568718005673873061
    INFO [03/19/21 16:39:21:81] Git tag '0.1.89': 0.1.89 with commit count source 0af6c896b58e56e424a1bb1fb7882fa19495af47
    INFO [03/19/21 16:39:21:81] Git tag '0.1.88': 0.1.88 with commit count source ed717bf034298df259524851c0595908e3d29061
    INFO [03/19/21 16:39:21:81] Git tag '0.1.87': 0.1.87 with commit count source c975e64efed6f9c2eb77afbe19016b68ddcb5434
    INFO [03/19/21 16:39:21:81] Git tag '0.1.86': 0.1.86 with commit count source 4e2b54fcd5124755daf82650f88e3d7c40120814
    INFO [03/19/21 16:39:21:81] Git tag '0.1.84': 0.1.84 with commit count source da098df362985b001f55db0c58ad3112c66f44c4
    INFO [03/19/21 16:39:21:81] Git tag '0.1.83': 0.1.83 with commit count source 6be11092289df22915967ad7b13e166f22acc6f7
    INFO [03/19/21 16:39:21:81] Git tag '0.1.82': 0.1.82 with commit count source 0bf538a6c48c9761055894ec20588652b87d667c
    INFO [03/19/21 16:39:21:81] Git tag '0.1.81': 0.1.81 with commit count source 8a4291c043dc5a2b576a87245ea6a2beed08a380
    INFO [03/19/21 16:39:21:81] Git tag '0.1.80': 0.1.80 with commit count source 5fe975ee3564bc03aab1556206fd484733b74831
    INFO [03/19/21 16:39:21:81] Git tag '0.1.79': 0.1.79 with commit count source 2bd38dc0734f5f19fef9f0296a3f85e995c5b3e9
    INFO [03/19/21 16:39:21:81] Git tag '0.1.78': 0.1.78 with commit count source c01ef21531e2668d3a53ecb0ffdf528ace7dda54
    INFO [03/19/21 16:39:21:81] Git tag '0.1.77': 0.1.77 with commit count source cbc5257677b649ea91b9f3548e4a010164b24384
    INFO [03/19/21 16:39:21:82] Git tag '0.1.76': 0.1.76 with commit count source f0a106cfcfed35966b52de08fc83b7c21e067502
    INFO [03/19/21 16:39:21:82] Git tag '0.1.75': 0.1.75 with commit count source 918d2fe76ea2eac757051507d5fed30b305591b5
    INFO [03/19/21 16:39:21:82] Git tag '0.1.74': 0.1.74 with commit count source 099acb70589a90ddccc514f7bc9f0308519baa7c
    INFO [03/19/21 16:39:21:82] Git tag '0.1.73': 0.1.73 with commit count source 7feac2e462501cfffa260c53f299cb9d441b2802
    INFO [03/19/21 16:39:21:82] Git tag '0.1.72': 0.1.72 with commit count source 987903cd4a75843cf344b4f35146a0d5aa47b10c
    INFO [03/19/21 16:39:21:82] Git tag '0.1.71': 0.1.71 with commit count source f89b04f6b14d80fdf9cfe16dfc199e01ff9b68b1
    INFO [03/19/21 16:39:21:82] Git tag '0.1.70': 0.1.70 with commit count source 933626a06bc0a41cef9c1b3bef229df453d7b78a
    INFO [03/19/21 16:39:21:82] Git tag '0.1.69': 0.1.69 with commit count source 3c505d7704718566c4936de5f71204725f9864c4
    INFO [03/19/21 16:39:21:82] Git tag '0.1.68': 0.1.68 with commit count source 0ffb8d159a84c9365f0034e4fb5b4636e24a17e8
    INFO [03/19/21 16:39:21:82] Git tag '0.1.67': 0.1.67 with commit count source 76c7f540a33d67764455e0ca862ed43304ddfae8
    INFO [03/19/21 16:39:21:82] Git tag '0.1.66': 0.1.66 with commit count source afc2741b9d4ce0d6db7f838cf8f226ea0be73359
    INFO [03/19/21 16:39:21:82] Git tag '0.1.65': 0.1.65 with commit count source be2456311bcb17fff6ea8bd175ba5afec4d60ad3
    INFO [03/19/21 16:39:21:82] Git tag '0.1.64': 0.1.64 with commit count source 2e837165f5e0b78135ce5cf33fcecb2196ed2f19
    INFO [03/19/21 16:39:21:82] Git tag '0.1.63': 0.1.63 with commit count source 9e1b3840fb61770852fc994298b98c7da367b590
    INFO [03/19/21 16:39:21:82] Git tag '0.1.62': 0.1.62 with commit count source 18bad45c0fe1c88a8e25fd6a0da1c956843126bf
    INFO [03/19/21 16:39:21:82] Git tag '0.1.61': 0.1.61 with commit count source c9357f3b7b67f3f6b5423827f6a21550d42d83b2
    INFO [03/19/21 16:39:21:82] Git tag '0.1.60': 0.1.60 with commit count source f58a9f3c50b7372b5ff11ebd8d788f94cf9b4de9
    INFO [03/19/21 16:39:21:82] Git tag '0.1.59': 0.1.59 with commit count source f1ce245066164725916d6a12c544bd459638a896
    INFO [03/19/21 16:39:21:82] Git tag '0.1.58': 0.1.58 with commit count source a017cdcb5f7a866480192b54f61580ee4bc977ff
    INFO [03/19/21 16:39:21:82] Git tag '0.1.57': 0.1.57 with commit count source 519bf2061b5b568f5a1b8e082898e63376c86db7
    INFO [03/19/21 16:39:21:82] Git tag '0.1.56': 0.1.56 with commit count source 1ec647af72b9cbd689b81d39f418a874e91bdd68
    INFO [03/19/21 16:39:21:82] Git tag '0.1.55': 0.1.55 with commit count source cc58df620a6ad05f9c7c9de2e5aa6828ac9f3d42
    INFO [03/19/21 16:39:21:82] Git tag '0.1.54': 0.1.54 with commit count source c0061a7673979a17b53efd5a63566ebdfa02c55b
    INFO [03/19/21 16:39:21:82] Git tag '0.1.53': 0.1.53 with commit count source 717be721f0e98d89ac7a0e5914b4e6f600e39423
    INFO [03/19/21 16:39:21:82] Git tag '0.1.52': 0.1.52 with commit count source 446d3684025acc0ea2c4ae2efa3bc2821dcf5dc2
    INFO [03/19/21 16:39:21:82] Git tag '0.1.51': 0.1.51 with commit count source f9ff29917f4b2e614e0a01f97238c186bd88b06c
    INFO [03/19/21 16:39:21:82] Git tag '0.1.50': 0.1.50 with commit count source 7f4896558a17e38b52267e372f116e0e6ff45652
    INFO [03/19/21 16:39:21:82] Git tag '0.1.49': 0.1.49 with commit count source f1712e9ffe5e43e9a77e243a92a0651c1480287a
    INFO [03/19/21 16:39:21:82] Git tag '0.1.48': 0.1.48 with commit count source 9c416b4df9bda7b33c568bf03c76f5b6fff16649
    INFO [03/19/21 16:39:21:82] Git tag '0.1.47': 0.1.47 with commit count source 9051565de2d7d7bd18aa02cf68462ae9b6a6f911
    INFO [03/19/21 16:39:21:82] Git tag '0.1.46': 0.1.46 with commit count source c7c15323686acb4ebb3135b79ba5d0016e838f77
    INFO [03/19/21 16:39:21:82] Git tag '0.1.45': 0.1.45 with commit count source 97cb7542e1f00cb1bd8bcc4f07716b64fc4fa5f6
    INFO [03/19/21 16:39:21:82] Git tag '0.1.44': 0.1.44 with commit count source 1513fde9b60562bfec281e09047ae590985eb2e7
    INFO [03/19/21 16:39:21:82] Git tag '0.1.43': 0.1.43 with commit count source f5084a21d8abcf7ac3a7810547099706983fb8e4
    INFO [03/19/21 16:39:21:82] Git tag '0.1.42': 0.1.42 with commit count source 0d2e776e6b70a7c758bc73bc33a410e192b7964b
    INFO [03/19/21 16:39:21:82] Git tag '0.1.41': 0.1.41 with commit count source 103133528abbb088f60a0c07ea4509b475e2bd36
    INFO [03/19/21 16:39:21:82] Git tag '0.1.40': 0.1.40 with commit count source afdf9d44a84da35e4d92ca289e5891a8c0ebe3ff
    INFO [03/19/21 16:39:21:82] Git tag '0.1.39': 0.1.39 with commit count source f8f8e19549f9173720d7cfe933bf39a34f3afbd0
    INFO [03/19/21 16:39:21:82] Git tag '0.1.38': 0.1.38 with commit count source 5e81398929adee2b80514464c13f6f195e66f43f
    INFO [03/19/21 16:39:21:82] Git tag '0.1.37': 0.1.37 with commit count source c7fd350a78f90d005e25dcc277aff4b1f0333066
    INFO [03/19/21 16:39:21:82] Git tag '0.1.36': 0.1.36 with commit count source 58bb7c96698a4ea42c28fea6880a9ee9e6739d20
    INFO [03/19/21 16:39:21:82] Git tag '0.1.35': 0.1.35 with commit count source e8462e1a91e380a78a353fb17383e369a35d906e
    INFO [03/19/21 16:39:21:82] Git tag '0.1.34': 0.1.34 with commit count source 882d59d4fd8eafd8f21c2c30d95907b93ad86e4f
    INFO [03/19/21 16:39:21:82] Git tag '0.1.33': 0.1.33 with commit count source 8bb2349606678c14f58b2dde908af22fb4bedfe9
    INFO [03/19/21 16:39:21:83] Git tag '0.1.32': 0.1.32 with commit count source a4d5b6bcca617b2297ae490630c9af20faf51dd5
    INFO [03/19/21 16:39:21:83] Git tag '0.1.31': 0.1.31 with commit count source 5476e3e2057e5e7d05256b314ba763f052d8f86a
    INFO [03/19/21 16:39:21:83] Git tag '0.1.30': 0.1.30 with commit count source 0c13cac1e9478f0d149062aa39e3111b545f7d98
    INFO [03/19/21 16:39:21:83] Git tag '0.1.29': 0.1.29 with commit count source 20b9edbc5597a8eb4d2b1762453f16c7fc0b7a87
    INFO [03/19/21 16:39:21:83] Git tag '0.1.28': 0.1.28 with commit count source f1653f14b33a9809660f982b131007bf5b1a27ae
    INFO [03/19/21 16:39:21:83] Git tag '0.1.26': 0.1.26 with commit count source 59a2d7616f59ac168e45bb2c31ba8274af4dd9e9
    INFO [03/19/21 16:39:21:83] Git tag '0.1.25': 0.1.25 with commit count source e949baea77c671416109df56cfca26cc58a16e6f
    INFO [03/19/21 16:39:21:83] Git tag '0.1.22': 0.1.22 with commit count source 590b329b0a2cfae2c70cd2605080e2894542f320
    INFO [03/19/21 16:39:21:83] Git tag '0.1.21': 0.1.21 with commit count source 3f65df2e0213015a6a6233efd5ab4e8f566427ce
    INFO [03/19/21 16:39:21:83] Git tag '0.1.20': 0.1.20 with commit count source dc2108ce72025a0bb7a050eebe89533b86ef0cf5
    INFO [03/19/21 16:39:21:83] Git tag '0.1.19': 0.1.19 with commit count source 3cd5f771685452b2779ce6b02077fc9e49844c14
    INFO [03/19/21 16:39:21:83] Git tag '0.1.18': 0.1.18 with commit count source fbc4b34422a9ded7c0a8226540b70caf3ff80524
    INFO [03/19/21 16:39:21:83] Git tag '0.1.17': 0.1.17 with commit count source f3e990912d7ecd78003ade497c3641425b8134d2
    INFO [03/19/21 16:39:21:83] Git tag '0.1.16': 0.1.16 with commit count source d1676a7f093809f8c387a5363532c62552c742f2
    INFO [03/19/21 16:39:21:83] Git tag '0.1.15': 0.1.15 with commit count source 8f16ae342ef073958bc27cfdc699abe97059d050
    INFO [03/19/21 16:39:21:83] Git tag '0.1.14': 0.1.14 with commit count source 566fb224d12e1416563d3d19efa8f8d5d3ffc0fc
    INFO [03/19/21 16:39:21:83] Git tag '0.1.13': 0.1.13 with commit count source e4c9a5dd7840f98bfb1eeba7b315f0eb2aaddb74
    INFO [03/19/21 16:39:21:83] Git tag '0.1.12': 0.1.12 with commit count source 12eb5b9306b5466732a41a314d03d6900a829938
    INFO [03/19/21 16:39:21:83] Git tag '0.1.11': 0.1.11 with commit count source c09c043540bd6bd7f2d27789d44fbdfdd750911c
    INFO [03/19/21 16:39:21:83] Git tag '0.1.10': 0.1.10 with commit count source 64ba9f1b346ec2f4acfa90f7661ebd8889dabfa3
    INFO [03/19/21 16:39:21:83] Git tag '0.1.9': 0.1.9 with commit count source f0f368f64c66e88780bafa4a0627bc622bcc53ee
    INFO [03/19/21 16:39:21:83] Git tag '0.1.8': 0.1.8 with commit count source fa9311c5e013c4a3b224c5386c044f2a4fa4f47d
    INFO [03/19/21 16:39:21:83] Git tag '0.1.7': 0.1.7 with commit count source b9dd1ed4945cf94b704e99b206fc243404206a8a
    INFO [03/19/21 16:39:21:83] Git tag '0.1.6': 0.1.6 with commit count source c9b83c370affefe9f745000ff3668f068439b950
    INFO [03/19/21 16:39:22:13] Found multiple base versions which will produce the same SemVer (0.1.319), taking oldest source for commit counting (Git tag '0.1.318')
    INFO [03/19/21 16:39:22:13] Base version used: Git tag '0.1.318': 0.1.318 with commit count source 0fec4df892ca3b0b0f7e7a002df7acf1cf0d3aba
  INFO [03/19/21 16:39:22:13] End: Calculating base versions (Took: 750.22ms)
  INFO [03/19/21 16:39:22:13] 4 commits found between 0fec4df Merged PR 1664: [S-77173] - remove SQL DB tags and eb45d5c Remove displayName tag from AAS to ensure version tags are not overwritten by blueprints
  INFO [03/19/21 16:39:22:13] Begin: Using mainline development mode to calculate current version
    INFO [03/19/21 16:39:22:14] Found possible mainline branches: master
    INFO [03/19/21 16:39:22:16] Mainline for current branch is master
    INFO [03/19/21 16:39:22:17] Found branch merge point; choosing 96e77c5 Merged PR 1685: {S-77173] Update to tagging of the Resource Groups with Deployment Id and Blueprint version as effective mainline tip
    INFO [03/19/21 16:39:22:17] Current branch (laje/S-77070-aas-tagging) was branch from 96e77c5 Merged PR 1685: {S-77173] Update to tagging of the Resource Groups with Deployment Id and Blueprint version
    INFO [03/19/21 16:39:22:17] Merge commit 96e77c5 Merged PR 1685: {S-77173] Update to tagging of the Resource Groups with Deployment Id and Blueprint version incremented base versions Patch, now 0.1.319
    INFO [03/19/21 16:39:22:17] 1 commits found between 96e77c5 Merged PR 1685: {S-77173] Update to tagging of the Resource Groups with Deployment Id and Blueprint version and eb45d5c Remove displayName tag from AAS to ensure version tags are not overwritten by blueprints
    INFO [03/19/21 16:39:22:17] Performing Patch increment for current branch
  INFO [03/19/21 16:39:22:18] End: Using mainline development mode to calculate current version (Took: 41.58ms)
  INFO [03/19/21 16:39:22:18] Using branch name to calculate version tag
  INFO [03/19/21 16:39:22:18] Begin: Getting version tags from branch 'refs/heads/laje/S-77070-aas-tagging'.
  INFO [03/19/21 16:39:22:30] End: Getting version tags from branch 'refs/heads/laje/S-77070-aas-tagging'. (Took: 123.26ms)
  INFO [03/19/21 16:39:22:31] Begin: Creating dictionary
  INFO [03/19/21 16:39:22:31] End: Creating dictionary (Took: 5.31ms)
  INFO [03/19/21 16:39:22:32] Begin: Storing version variables to cache file C:\Code\src\.git\gitversion_cache\60B67F852C73D243B79D5D48A01600B3732665D7.yml
  INFO [03/19/21 16:39:22:34] End: Storing version variables to cache file C:\Code\src\.git\gitversion_cache\60B67F852C73D243B79D5D48A01600B3732665D7.yml (Took: 29.41ms)
{
  "Major": 0,
  "Minor": 1,
  "Patch": 320,
  "PreReleaseTag": "laje-S-77070-aas-tagging.1",
  "PreReleaseTagWithDash": "-laje-S-77070-aas-tagging.1",
  "PreReleaseLabel": "laje-S-77070-aas-tagging",
  "PreReleaseLabelWithDash": "-laje-S-77070-aas-tagging",
  "PreReleaseNumber": 1,
  "WeightedPreReleaseNumber": 1,
  "BuildMetaData": null,
  "BuildMetaDataPadded": "",
  "FullBuildMetaData": "Branch.laje-S-77070-aas-tagging.Sha.eb45d5c6891e2b6a7c83b82b2f559ae05e136670",
  "MajorMinorPatch": "0.1.320",
  "SemVer": "0.1.320-laje-S-77070-aas-tagging.1",
  "LegacySemVer": "0.1.320-laje-S-77070-aas-ta1",
  "LegacySemVerPadded": "0.1.320-laje-S-77070-aas0001",
  "AssemblySemVer": "0.1.320.0",
  "AssemblySemFileVer": "0.1.320.0",
  "FullSemVer": "0.1.320-laje-S-77070-aas-tagging.1",
  "InformationalVersion": "0.1.320-laje-S-77070-aas-tagging.1+Branch.laje-S-77070-aas-tagging.Sha.eb45d5c6891e2b6a7c83b82b2f559ae05e136670",
  "BranchName": "laje/S-77070-aas-tagging",
  "EscapedBranchName": "laje-S-77070-aas-tagging",
  "Sha": "eb45d5c6891e2b6a7c83b82b2f559ae05e136670",
  "ShortSha": "eb45d5c",
  "NuGetVersionV2": "0.1.320-laje-s-77070-aas0001",
  "NuGetVersion": "0.1.320-laje-s-77070-aas0001",
  "NuGetPreReleaseTagV2": "laje-s-77070-aas0001",
  "NuGetPreReleaseTag": "laje-s-77070-aas0001",
  "VersionSourceSha": "96e77c5c39b47d6dacf8e27a3a4db086d82b59c9",
  "CommitsSinceVersionSource": 1,
  "CommitsSinceVersionSourcePadded": "0001",
  "UncommittedChanges": 12,
  "CommitDate": "2021-03-19"
}
  INFO [03/19/21 16:39:22:40] Done writing
```

Output from 5.6.3:
```
INFO [03/22/21 9:39:04:33] Working directory: C:\Code\src
INFO [03/22/21 9:39:04:36] Project root is: C:\Code\src\
INFO [03/22/21 9:39:04:36] DotGit directory is: C:\Code\src\.git
INFO [03/22/21 9:39:04:38] Begin: Loading version variables from disk cache
  INFO [03/22/21 9:39:04:38] Begin: Deserializing version variables from cache file C:\Code\src\.git\gitversion_cache\60B67F852C73D243B79D5D48A01600B3732665D7.yml
  INFO [03/22/21 9:39:04:42] End: Deserializing version variables from cache file C:\Code\src\.git\gitversion_cache\60B67F852C73D243B79D5D48A01600B3732665D7.yml (Took: 40.57ms)
  INFO [03/22/21 9:39:04:42] End: Loading version variables from disk cache (Took: 42.31ms)
{
  "Major": 0,
  "Minor": 1,
  "Patch": 320,
  "PreReleaseTag": "laje-S-77070-aas-tagging.1",
  "PreReleaseTagWithDash": "-laje-S-77070-aas-tagging.1",
  "PreReleaseLabel": "laje-S-77070-aas-tagging",
  "PreReleaseLabelWithDash": "-laje-S-77070-aas-tagging",
  "PreReleaseNumber": 1,
  "WeightedPreReleaseNumber": 1,
  "BuildMetaData": "",
  "BuildMetaDataPadded": "",
  "FullBuildMetaData": "Branch.laje-S-77070-aas-tagging.Sha.eb45d5c6891e2b6a7c83b82b2f559ae05e136670",
  "MajorMinorPatch": "0.1.320",
  "SemVer": "0.1.320-laje-S-77070-aas-tagging.1",
  "LegacySemVer": "0.1.320-laje-S-77070-aas-ta1",
  "LegacySemVerPadded": "0.1.320-laje-S-77070-aas0001",
  "AssemblySemVer": "0.1.320.0",
  "AssemblySemFileVer": "0.1.320.0",
  "FullSemVer": "0.1.320-laje-S-77070-aas-tagging.1",
  "InformationalVersion": "0.1.320-laje-S-77070-aas-tagging.1+Branch.laje-S-77070-aas-tagging.Sha.eb45d5c6891e2b6a7c83b82b2f559ae05e136670",
  "BranchName": "laje/S-77070-aas-tagging",
  "EscapedBranchName": "laje-S-77070-aas-tagging",
  "Sha": "eb45d5c6891e2b6a7c83b82b2f559ae05e136670",
  "ShortSha": "eb45d5c",
  "NuGetVersionV2": "0.1.320-laje-s-77070-aas0001",
  "NuGetVersion": "0.1.320-laje-s-77070-aas0001",
  "NuGetPreReleaseTagV2": "laje-s-77070-aas0001",
  "NuGetPreReleaseTag": "laje-s-77070-aas0001",
  "VersionSourceSha": "96e77c5c39b47d6dacf8e27a3a4db086d82b59c9",
  "CommitsSinceVersionSource": 1,
  "CommitsSinceVersionSourcePadded": "0001",
  "UncommittedChanges": 12,
  "CommitDate": "2021-03-19"
}
  INFO [03/22/21 9:39:04:48] Done writing
```

--- a/src/tasks/gitversion/main.ts
+++ b/src/tasks/gitversion/main.ts
@@ -42,8 +42,8 @@ export async function run() {
         const result = await gitVersionTool.run(settings)
         const { stdout } = result
         const jsonOutput = stdout.substring(
-            stdout.indexOf('{'),
-            stdout.indexOf('}') + 1
+            stdout.lastIndexOf('{'),
+            stdout.lastIndexOf('}') + 1
         )
 
         const gitversion = JSON.parse(jsonOutput) as GitVersionOutput

